tools/chapel-py/setup.py

```diff
@@ -24,7 +24,7 @@
 import sys
 import glob
 
-chpl_home = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+chpl_home = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
could you do something like `pathlib.Path(..) / ".." / ".." / ".."`? :D

Reply:
I think so? I believe pathlib is now widely supported, I need to get into the habit of using it